Jenkinsfile

```diff
@@ -206,7 +206,12 @@ pipeline {
                     fingerprintArtifacts: true
                 )
                 sh 'kubectl get nodes -o wide'
-                sh "nix-shell --run './scripts/e2e-test.sh --device /dev/sdb --tag \"${env.GIT_COMMIT_SHORT}\" --registry \"${env.REGISTRY}\"'"
+                sh "nix-shell --run './scripts/e2e-test.sh --device /dev/sdb --tag \"${env.GIT_COMMIT_SHORT}\" --registry \"${env.REGISTRY}\" --logsdir ./logs/mayastor --logs '"
```

Comment:
This is a comment about logging in general. From usability perspective it seems that having both options, logsdir and logs, does not make sense. Most of the time I want either the logs to be generated in which case I specify logsdir or I don't want the logs in which case I run it without the option. I mean, who would like to see all logs printed to stdout if we can save them to files now. Is there such use case?

Reply:
The semantics of --logs is `always generate logs` , if not specified logs are only generated on test failure.
--logsdir specifies where the log files (if any) are the be generated.